source/user-manual/user-administration/password-management.rst

```diff
@@ -120,7 +120,7 @@ To generate and change passwords for all the Wazuh indexer users, run the script
     INFO: The password for user logstash is SUbk4KTmLl*geQbUg0c5tyfwahjDMhx5
     INFO: The password for user readall is ?w*Itj1Lgz.5w.C7vOw0Kxi7G94G8bG*
     INFO: The password for user snapshotrestore is Z6UXgM8Sr0bfV.i*6yPPEUY3H6Du2rdz
-    WARNING: Wazuh indexer passwords changed. Remember to update the password in the Wazuh dashboard, Wazuh server, and Filebeat nodes if necessary, and restart the services.
+    WARNING: Wazuh indexer passwords changed. Remember to update the password in the Wazuh dashboard, Wazuh server, keystore tool, and Filebeat nodes if necessary, and restart the services.
```

Comment:
```suggestion
    WARNING: Wazuh indexer passwords changed. Remember to update the password in the Wazuh dashboard, Wazuh server, Wazuh server keystore using wazuh-keystore tool, and Filebeat nodes if necessary, and restart the services.
```

Reply:
Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/166a6616aa43bcfd53cffd26dd40a3764a21f901) commit